app/services/encrypted_key_maker.rb

```diff
@@ -83,7 +83,11 @@ def looks_like_kms?(user_access_key, encryption_key)
   end
 
   def aws_client
-    @_aws_client ||= Aws::KMS::Client.new(region: Figaro.env.aws_region)
+    @_aws_client ||= Aws::KMS::Client.new(
+      region: Figaro.env.aws_region,
+      instance_profile_credentials_timeout: 5, # defaults to 1 second
```

Comment:
How did you decide on 5 seconds? That may be a little long. This is used in a bunch of places and will block a request thread while it is waiting.

Reply:
No particular reason. It was the example provided in the GitHub issue I linked to above. I'm happy to change it. What do you think is a reasonable value, and how many times should we retry?

Reply:
I am not sure. Maybe stick with the default timeout but w/ 5 retries?

Reply:
OK, will do.